### PR TITLE
update governance

### PR DESCRIPTION
**Pull request by drammock**
Update our public governance document.

TODOs:

- [x] now that new governance is adopted, set a deadline for current maintainer team members to volunteer for steering council.  *deadline is Nov 1 and was announced on Discord and maintainer email list*
- [x] Once steering council membership finalized, update `doc/overview/people.rst`
- [x] the sections on "the project" and "institutional partners" were not included in the proposed restructuring. For now I've left them in place. If folks disagree with that course of action, I think we could summarize the "institutional partners" content at the start of the "institutional partners" list on `doc/overview/people.rst` instead.

### CODE REVIEW

**Review by agramfort — APPROVED** [review]

File: doc/development/governance.rst [hunk 1]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+  multiple submodules), in partnership with the maintainer team
+- Infrastructure decisions (e.g., dependency version policy, release cadence, CI
+  management, etc), in partnership with the maintainer team
+- Any other Governance task not mentioned elsewhere, and that falls outside of the
+  responsibilities of other teams
+- Attendance at Steering Council meetings (approx. every 2 weeks; time to be decided
+  among SC members)
+- Attendance at advisory board meetings (approx. every 1-2 years)
+- Write funding proposals
+- Communicate/coordinate with maintainer team
```
Comment (britta-wstnr) [inline doc/development/governance.rst:94]:
```suggestion
- Communicate/coordinate with Maintainer Team
```

File: doc/development/governance.rst [hunk 2]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+- Coordination with the larger Scientific Python ecosystem
+- Large-scale changes to the software (e.g., type hints, docdict, things that affect
+  multiple submodules), in partnership with the maintainer team
+- Infrastructure decisions (e.g., dependency version policy, release cadence, CI
+  management, etc), in partnership with the maintainer team
+- Any other Governance task not mentioned elsewhere, and that falls outside of the
+  responsibilities of other teams
+- Attendance at Steering Council meetings (approx. every 2 weeks; time to be decided
+  among SC members)
+- Attendance at advisory board meetings (approx. every 1-2 years)
```
Comment (britta-wstnr) [inline doc/development/governance.rst:92]:
```suggestion
- Attendance at Advisory Board meetings (approx. every 1-2 years)
```

File: doc/development/governance.rst [hunk 3]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
-respond — at least one week.
-
-In practice, we anticipate that for most Steering Council decisions
-(e.g., voting in new members) a more informal process will suffice.
+The Steering Council is responsible for guiding and shepherding the project on a
+day-to-day basis. Duties:
+
+- Obtaining funding (either by writing grants specifically for MNE development, or
+  convincing others to include funds for MNE development in their research grants)
+- Translating high-level roadmap guidance from the advisory board (e.g. “better support
```
Comment (britta-wstnr) [inline doc/development/governance.rst:79]:
```suggestion
- Translating high-level roadmap guidance from the Advisory Board (e.g. “better support
```

File: doc/development/governance.rst [hunk 4]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+
+Leadership Roles
+^^^^^^^^^^^^^^^^
+
+The MNE-Python leadership structure shall consist of:
+
+Maintainer Team
+---------------
+
+The maintainer team is responsible for implementing changes to the software and
```
Comment (britta-wstnr) [inline doc/development/governance.rst:54]:
```suggestion
The Maintainer Team is responsible for implementing changes to the software and
```

File: doc/development/governance.rst [hunk 5]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+-------------------------
+
+The Steering Council will have mandatory meetings every two weeks to discuss project
+management and funding. The Steering Council may decide to change meeting time or
+frequency at their discretion.
+
+All-hands meetings
+------------------
+
+At least once a year, all Maintainers and Steering Council members should come together
```
Comment (britta-wstnr) [inline doc/development/governance.rst:140]:
```suggestion
At least once a year, all maintainers and Steering Council members should come together
```

File: doc/development/governance.rst [hunk 6]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+- Communicating important decisions (and the context for why those decisions were
+  taken) to the community
+
+External Advisory Board
+-----------------------
+
+The External Advisory Board is responsible for high-level roadmap and funding
+guidance. Duties:
+
+- Attendance at advisory board meetings (approx. every 1-2 years)
```
Comment (britta-wstnr) [inline doc/development/governance.rst:117]:
```suggestion
- Attendance at Advisory Board meetings (approx. every 1-2 years)
```

File: doc/development/governance.rst [hunk 7]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+maintainer team, *ex officio*, and thus shall have the necessary rights and privileges
+afforded to maintainers (passwords, merge rights, etc).
+
+Chair of the Steering Council
+-----------------------------
+
+The Chair of the Steering Council is responsible for liaising between the Steering
+Council and the community. Duties:
+
+- Convening the steering council meetings
```
Comment (britta-wstnr) [inline doc/development/governance.rst:106]:
```suggestion
- Convening the Steering Council meetings
```

File: doc/development/governance.rst [hunk 8]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+- Any other Governance task not mentioned elsewhere, and that falls outside of the
+  responsibilities of other teams
+- Attendance at Steering Council meetings (approx. every 2 weeks; time to be decided
+  among SC members)
+- Attendance at advisory board meetings (approx. every 1-2 years)
+- Write funding proposals
+- Communicate/coordinate with maintainer team
+
+Members of the Steering Council shall additionally be considered as members of the
+maintainer team, *ex officio*, and thus shall have the necessary rights and privileges
```
Comment (britta-wstnr) [inline doc/development/governance.rst:97]:
```suggestion
Maintainer Team, *ex officio*, and thus shall have the necessary rights and privileges
```

File: doc/development/governance.rst [hunk 9]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+- Attendance at advisory board meetings (approx. every 1-2 years)
+- Periodically communicating with Steering Council to impart guidance
+
+Meetings
+^^^^^^^^
+
+Maintainer meetings
+-------------------
+
+The maintainer team can decide if there should be maintainer meetings or not. These
```
Comment (britta-wstnr) [inline doc/development/governance.rst:126]:
```suggestion
The Maintainer Team can decide if there should be maintainer meetings or not. These
```

File: doc/development/governance.rst [hunk 10]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+- Ad-hoc additions are possible by initiative of the steering council for exceptional
+  circumstances, e.g., hiring someone with grant funds to specifically do MNE
+  maintenance. These ad-hoc additions do not necessarily require a vote from the
+  maintainer team, as the job application procedures of the hiring institution are
+  assumed to be sufficiently rigorous to avoid bias, nepotism, etc.
+
+Duration of service
+~~~~~~~~~~~~~~~~~~~
+
+Appointment to the maintainer team is for an indefinite term.
```
Comment (britta-wstnr) [inline doc/development/governance.rst:167]:
```suggestion
Appointment to the Maintainer Team is for an indefinite term.
```

File: doc/development/governance.rst [hunk 11]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+circumstances:
+
+- *Voluntary resignation*, at any time, for any reason.
+- *Inactivity*. Once per year, before the annual all-hands meeting, the Steering Council
+  shall assess maintainer activity for the preceding year. Any maintainers seeming to be
+  inactive shall be contacted and given opportunity to dispute their inactivity (e.g.,
+  by highlighting ways they have been working in the MNE community that may not be
+  visible from simple metrics like GitHub or forum activity reports). Maintainers who do
+  not dispute their inactivity (or fail to respond within 14 days, or longer at the
+  discretion of the Steering Council) shall be removed from the maintainer team. In
```
Comment (britta-wstnr) [inline doc/development/governance.rst:183]:
```suggestion
  discretion of the Steering Council) shall be removed from the Maintainer Team. In
```

File: doc/development/governance.rst [hunk 12]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+
+- *Voluntary resignation*, at any time, for any reason.
+- *Conduct*. At any time, a member of the Steering Council may be removed by unanimous
+  vote of the code-of-conduct committee, for violations of our community guidelines (in
+  accordance with the enforcement guidelines outlined therein).
+
+External Advisory Board
+-----------------------
+
+The external advisory board shall be populated by invitation from the Steering Council.
```
Comment (britta-wstnr) [inline doc/development/governance.rst:241]:
```suggestion
The External Advisory Board shall be populated by invitation from the Steering Council.
```

File: doc/development/governance.rst [hunk 13]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+with a threshold of 50% + 1 vote. That means, everyone ranks those candidates (in order
+of preference) that they could see filling the role in question. Note that it is
+possible for a voter to reject all candidates by submitting a blank ballot, so that if a
+single person is running for a seat it is still possible for them to fail to be elected
+if enough voters cast blank ballots.
+
+Voting for the steering council
+-------------------------------
+
+Votes for steering council membership shall be scheduled as-needed to address steering
```
Comment (britta-wstnr) [inline doc/development/governance.rst:271]:
```suggestion
Votes for Steering Council membership shall be scheduled as-needed to address Steering
```

File: doc/development/governance.rst [hunk 14]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+council workload, and advertised to eligible candidates (i.e., the maintainer team) for
```
Comment (britta-wstnr) [inline doc/development/governance.rst:272]:
```suggestion
Council workload, and advertised to eligible candidates (i.e., the Maintainer Team) for
```

File: doc/development/governance.rst [hunk 15]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+a minimum of 14 days, after which a vote of current maintainers and steering council
```
Comment (britta-wstnr) [inline doc/development/governance.rst:273]:
```suggestion
a minimum of 14 days, after which a vote of current maintainers and Steering Council
```

File: doc/development/governance.rst [hunk 16]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+
+Voting for the maintainer team
+------------------------------
+
+Votes for additions to the maintainer team shall be scheduled promptly following the
+annual all-hands meeting. The maintainer team shall advertise the open seats via online
+MNE-Python channels. Applications (consisting of a short candidate statement) must be
+open for a minimum of 14 days, after which a vote of the current maintainers and
+steering council shall be scheduled. The maintainer team shall set up a confidential
+submission system for applications (consisting of short candidate statements), such as a
```
Comment (britta-wstnr) [inline doc/development/governance.rst:284]:
```suggestion
Steering Council shall be scheduled. The Maintainer Team shall set up a confidential
submission system for applications (consisting of short candidate statements), such as a
```

File: doc/development/governance.rst [hunk 17]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+supporting the user community. Duties:
+
+- Infrastructure/large-scale software decisions, in partnership with the steering
+  council
+- Reviewing and merging pull requests
+- Responding to issues on GitHub
+- Monitoring CI failures and addressing them
+- Community maintenance: answering forum posts, holding office hours
+- Community information: social media announcements (releases, new features, etc)
+- Training new members of the maintainer team
```
Comment (britta-wstnr) [inline doc/development/governance.rst:64]:
```suggestion
- Training new members of the Maintainer Team
```

File: doc/development/governance.rst [hunk 18]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+The MNE-Python leadership structure shall consist of:
+
+Maintainer Team
+---------------
+
+The maintainer team is responsible for implementing changes to the software and
+supporting the user community. Duties:
+
+- Infrastructure/large-scale software decisions, in partnership with the steering
+  council
```
Comment (britta-wstnr) [inline doc/development/governance.rst:58]:
```suggestion
- Infrastructure/large-scale software decisions, in partnership with the Steering
  Council
```

File: doc/development/governance.rst [hunk 19]
```diff
@@ -39,214 +38,280 @@ Contributors and Users as low as possible.
...
+- Translating high-level roadmap guidance from the advisory board (e.g. “better support
+  for OPMs”) into actionable roadmap items (e.g., “Add support for OPM manufacturers
+  besides QuSpin, and add standard preprocessing routines for coreg and OPM-specific
+  artifacts”)
+- Coordination with the larger Scientific Python ecosystem
+- Large-scale changes to the software (e.g., type hints, docdict, things that affect
+  multiple submodules), in partnership with the maintainer team
+- Infrastructure decisions (e.g., dependency version policy, release cadence, CI
+  management, etc), in partnership with the maintainer team
+- Any other Governance task not mentioned elsewhere, and that falls outside of the
```
Comment (britta-wstnr) [inline doc/development/governance.rst:88]:
```suggestion
- Any other governance task not mentioned elsewhere, and that falls outside of the
```

**Review by britta-wstnr — COMMENTED** [review]
LGTM.
Unified capitalization a bit more - hope I have caught all (most) things.

**Review by larsoner — APPROVED** [review]

**Comment by sappelhoff** [comment]
> the sections on "the project" and "institutional partners" were not included in the proposed restructuring. For now I've left them in place.

I agree with this. However note that I have not been employed by "Max Planck Institute für Bildungsforschung" since 2024-07-14. So I guess this triggered the "1 year grace period" for them until they are moved to "former partners". Shall I create an event for this on our mne steering council calendar?

**Review by sappelhoff — APPROVED** [review]

**Review by larsoner — APPROVED** [review]
Marking for merge-when-green, thanks in advance @drammock !

**Pull request merged**